ui/Modal/ManageRemotes/Peer.svelte

```diff
@@ -0,0 +1,66 @@
+<script lang="ts">
```

Comment:
And this directory `ManagePeers/*`?

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1014/commits/8e5e847a1b0d3569a02b867b97c9b60a578f7f19